### DISCUSSION TITLE
Delete item - how does it work?

### DISCUSSION THREAD

**Discussion by tomas-zivotsky** [C1rdec/PoeLurker#397] [Q&A] [answered]
There is a feature 'Delete item'. I haven't found any guide how to use it. Is there any hotkey to trigger it? if so, can it be remapped? Do I need to have the item picked up or the hover is enough?

![image](https://github.com/C1rdec/Poe-Lurker/assets/53608296/62f8356e-c535-45c4-9dc9-a84fc5ccd407)

Thanks for the amazing job.

**Comment by C1rdec** [comment, marked as answer]
It's the 'Del' key on the keyboard and you only need to hover the item. Poe Lurker will perform a Click and the chat command.

**Comment by Supsie** [comment]
hello, I just uninstalled the damn app, I have just calmed down enough not to be writing rudely, but seriously having the delete items by default  it is very dumb and moronic  decision you have made. i lost two belts, guess what was one of the belts that got deleted?

for future reference options that can make items go poof should not be on by default specially is there is no fucking promt after it.

very good and useful app but this del to delet items is very stupid specially if the del is right next to enter key and can be pressed accidentally.

**Comment by C1rdec** [comment]
Delete Item is `false` by default...
You enabled it.
